geps/gep-1713/index.md

```diff
@@ -0,0 +1,433 @@
+# GEP-1713: Standard Mechanism to Merge Multiple Gateways
+
+* Issue: [#1713](/kubernetes-sigs/gateway-api/issues/1713)
+* Status: Provisional
+
+(See status definitions [here](overview.md#status).)
+
+## tl;dr
+
+The `Gateway` Resource is a point of contention since it is the only place to attach listeners with certificates. We propose a new resource called `ListenerSet` to allow a shared list of listeners to be attached to a single `Gateway`.
+
+## Goals
+- Define a mechanism to merge listeners into a single `Gateway`
+
+## Future Goals (Beyond the GEP)
+- Attaching listeners to `Gateways` in different namespaces
```

Comment:
Recommend adding some of the goals/concepts from the [hierarchy doc](https://docs.google.com/document/d/1qj7Xog2t2fWRuzOeTsWkabUaVeOF7_2t_7appe8EXwA/edit#heading=h.w311n4l5qmwk) here.